pixel/head.html

```diff
@@ -0,0 +1,20 @@
+<script>
+  partytown = {
+    forward: ['dataLayer.push', 'vtexrca'].concat("{{settings.forward}}".split(',')),
+    lib: '/_v/public/assets/v1/published/vtex.partytown@0.0.1-alpha.10/public/~partytown/',
+    resolveUrl: function (url) {
+      var domain = url.hostname.replace('www.', '')
+
+      if ("{{settings.toProxyDomains}}".split(',').indexOf(domain) >= 0) {
+        // TODO: This URL is just for test purposes and must be removed before deploying
+        return new URL(`https://reverse-proxy-vtex.herokuapp.com/${url}`);
+      }
+
+      return url
+    },
+  };
```

Comment:
```suggestion
partytown={forward:["dataLayer.push","vtexrca"].concat(["{{settings.forward}}"]),lib:`/_v/public/assets/v1/published/vtex.partytown@${process.env.VTEX_APP_ID}/public/~partytown/`,resolveUrl:function(a){var b=a.hostname.replace("www.","");return 0<=["{{settings.toProxyDomains}}"].indexOf(b)?new URL(`https://reverse-proxy-vtex.herokuapp.com/${a}`):a}};
```

Reply:
I'll minify it. Wasn't builder-hub suppose to do this?

Reply:
> Also, would be good if you add an script that update or generate the partytownSnippet used at pixel/head.html

I created the `build` script.

It will minify the file and append the snippet to the end of it